notebooks/SMAP/2.0 Read and Plot SMAP data.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    print(type(ret_flag_L3_P))
</pre><p>I am not sure this is necessary.  All Dataset data arrays are <code>numpy.ndarray</code> and this is covered when you read the soil moisture variable. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>